src/common/load-plugins.js

```diff
@@ -113,6 +142,20 @@ async function findPluginsInNodeModules(nodeModulesDir) {
   return pluginPackageJsonPaths.map(path.dirname);
 }
 
+async function findPluginDirsInPnpmVirtualStore(searchDirectory) {
+  const pluginPackageJsonPaths = await fastGlob(
+    ["prettier-plugin-*", "@*/node_modules"],
```

Comment:
Shouldn't plugins also stored like `prettier-plugin-foo@1.0.0`? I don't know how the `@*/plugin-*` stored, but are you sure these patterns correct?

Reply:
In the pnpm virtual store, the three possible types of plugins (`@prettier/plugin-*`, and `@<name>/prettier-plugin-*`, `prettier-plugin-*`) are stored like this:

```
.pnpm/
├── @prettier+plugin-foo@<version>/
│   └── node_modules/
│       └── @prettier/
│           └── plugin-foo/
├── @<name>+prettier-plugin-bar@<version>/
│   └── node_modules/
│       └── @<name>/
│           └── prettier-plugin-bar/
└── prettier-plugin-baz@<version>/
    └── node_modules/
        └── prettier-plugin-baz/
```

I don't think the `+package-name` part of the directory used to be there, but now that it is, I can just have the patterns be `["prettier-plugin-*", "@prettier+plugin-*", "@*+prettier-plugin-*"]`.